### PR TITLE
Bundle reference named pipeline fix and query params fix

### PR DESCRIPTION
**Pull request by DeMoorJasper**
# ↪️ Pull Request

This PR is based on #5219 with additional bugfixes and tests for `getURLReplacement`

## ✔️ PR Todo

- [x] Added/updated unit tests for this change
- [x] Filled out test instructions (In case there aren't any unit tests)
- [x] Included links to related issues/PRs

### CODE REVIEW

File: packages/core/utils/src/replaceBundleReferences.js [hunk 1]
```diff
@@ -155,17 +155,20 @@ function getURLReplacement({
...
-  let url = URL.parse(dependency.moduleSpecifier);
+|}): {|from: string, to: string|} {
   let to;
+
   if (relative) {
-    url.pathname = relativeBundlePath(fromBundle, toBundle, {
-      leadingDotSlash: false,
-    });
-    to = URL.format(url);
+    to = URL.format(
```
Comment (DeMoorJasper) [inline packages/core/utils/src/replaceBundleReferences.js:162]: Is the URL.format and URL.parse really necessary?

**Comment by parcel-benchmark** [comment]
## Benchmark Results
<details><summary>Kitchen Sink ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 7.05s | -704.00ms 🚀 |
| Cached | 3.08s | -78.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 272.00ms | +44.00ms ⚠️ |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 216.00ms | -225.00ms 🚀 |
| dist/legacy/index.45cccf70.js | 1.10kb | +0.00b | 2.32s | -158.00ms 🚀 |
| dist/modern/index.1964d1f6.js | 1.10kb | +0.00b | 2.28s | -378.00ms 🚀 |
| dist/legacy/index.html | 701.00b | +0.00b | 2.44s | -157.00ms 🚀 |
| dist/modern/index.html | 701.00b | +0.00b | 2.38s | -367.00ms 🚀 |
| dist/legacy/index.b3acc8eb.css | 77.00b | +0.00b | 2.28s | -184.00ms 🚀 |
| dist/modern/index.7bc22a8b.css | 77.00b | +0.00b | 2.26s | -367.00ms 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 343.00ms | -39.00ms 🚀 |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 206.00ms | -105.00ms 🚀 |
| dist/legacy/index.45cccf70.js | 1.10kb | +0.00b | 408.00ms | -56.00ms 🚀 |
| dist/modern/index.1964d1f6.js | 1.10kb | +0.00b | 279.00ms | -130.00ms 🚀 |
| dist/legacy/index.html | 701.00b | +0.00b | 408.00ms | -57.00ms 🚀 |
| dist/modern/index.html | 701.00b | +0.00b | 279.00ms | -130.00ms 🚀 |
| dist/legacy/index.b3acc8eb.css | 77.00b | +0.00b | 408.00ms | -56.00ms 🚀 |
| dist/modern/index.7bc22a8b.css | 77.00b | +0.00b | 279.00ms | -130.00ms 🚀 |

</p></details>

<details><summary>React HackerNews ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 24.72s | +523.00ms |
| Cached | 2.94s | -33.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.js | 478.81kb | +0.00b | 11.37s | +878.00ms ⚠️ |
| dist/PermalinkedComment.9c5ad065.js | 4.22kb | +0.00b | 6.01s | +409.00ms ⚠️ |
| dist/UserProfile.c1927888.js | 1.62kb | +0.00b | 6.01s | +409.00ms ⚠️ |
| dist/NotFound.f8a0616d.js | 387.00b | +0.00b | 899.00ms | +183.00ms ⚠️ |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 124.00ms | -24.00ms 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.js | 478.81kb | +0.00b | 206.00ms | +18.00ms ⚠️ |
| dist/PermalinkedComment.9c5ad065.js | 4.22kb | +0.00b | 206.00ms | +18.00ms ⚠️ |
| dist/UserProfile.c1927888.js | 1.62kb | +0.00b | 206.00ms | +18.00ms ⚠️ |
| dist/NotFound.f8a0616d.js | 387.00b | +0.00b | 211.00ms | -28.00ms 🚀 |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 125.00ms | +15.00ms ⚠️ |

</p></details>

<details><summary>AtlasKit Editor ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 2.87m | +3.98s |
| Cached | 7.63s | +776.00ms ⚠️ |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.8016b423.js | 2.72mb | -3.00b 🚀 | 1.56m | +700.00ms |
| dist/pdfRenderer.15a6a023.js | 1.11mb | +3.00b ⚠️ | 49.39s | +1.98s |
| dist/popup.6c453e3c.js | 169.96kb | +0.00b | 29.60s | -17.60s 🚀 |
| dist/EmojiPickerComponent.ae601247.js | 75.29kb | +0.00b | 10.31s | +1.76s ⚠️ |
| dist/media-viewer.7079364e.js | 72.03kb | +0.00b | 14.64s | +1.78s ⚠️ |
| dist/card.645a5c17.js | 55.01kb | +0.00b | 12.16s | +1.79s ⚠️ |
| dist/Modal.19ffed32.js | 41.27kb | +0.00b | 6.43s | +1.85s ⚠️ |
| dist/component.d7c29980.js | 30.70kb | +0.00b | 5.48s | +735.00ms ⚠️ |
| dist/component.54080a26.js | 22.50kb | +0.00b | 6.42s | +1.85s ⚠️ |
| dist/DatePicker.0df05dac.js | 21.21kb | +0.00b | 10.31s | +1.75s ⚠️ |
| dist/EmojiPickerComponent.7dc0ebd0.js | 18.23kb | +0.00b | 10.30s | +1.75s ⚠️ |
| dist/js.5733a017.js | 16.40kb | +0.00b | 6.42s | +1.85s ⚠️ |
| dist/ui.fee77548.js | 13.97kb | +0.00b | 12.16s | +1.80s ⚠️ |
| dist/workerHasher.1e706e77.js | 11.75kb | +0.00b | 12.16s | +1.80s ⚠️ |
| dist/component.77c0f682.js | 6.05kb | +0.00b | 7.89s | +2.93s ⚠️ |
| dist/card.383bd9bd.js | 5.61kb | +0.00b | 14.56s | +1.72s ⚠️ |
| dist/media-viewer.59c17dad.js | 3.92kb | +0.00b | 14.56s | +1.72s ⚠️ |
| dist/EmojiPickerComponent.627aca6b.js | 3.58kb | +0.00b | 10.31s | +2.23s ⚠️ |
| dist/png-chunks-extract.a844fdb6.js | 3.41kb | +0.00b | 6.42s | +1.85s ⚠️ |
| dist/dropzone.3c67f0a6.js | 3.23kb | +0.00b | 27.56s | +1.93s ⚠️ |
| dist/Modal.0304b9ed.js | 3.08kb | +0.00b | 6.42s | +1.85s ⚠️ |
| dist/16.290b4a43.js | 2.38kb | +0.00b | 3.17s | +1.68s ⚠️ |
| dist/card.eea6ad8c.js | 2.03kb | +0.00b | 27.55s | +1.92s ⚠️ |
| dist/ResourcedEmojiComponent.abe8596c.js | 2.00kb | +0.00b | 5.43s | +800.00ms ⚠️ |
| dist/date.231500d2.js | 1.73kb | +0.00b | 7.10s | +1.63s ⚠️ |
| dist/16.d16f14ed.js | 1.73kb | +0.00b | 6.80s | +1.64s ⚠️ |
| dist/images.80c92f5e.js | 1.67kb | +0.00b | 9.11s | +3.50s ⚠️ |
| dist/16.1b6ca79f.js | 1.66kb | +0.00b | 6.61s | +1.65s ⚠️ |
| dist/feedback.d5590b34.js | 1.64kb | +0.00b | 7.54s | -1.02s 🚀 |
| dist/list-number.c35f9589.js | 1.46kb | +0.00b | 7.22s | +1.62s ⚠️ |
| dist/status.8ab4226b.js | 1.46kb | +0.00b | 7.40s | +1.62s ⚠️ |
| dist/code.3dd71280.js | 1.38kb | +0.00b | 7.10s | +1.63s ⚠️ |
| dist/16.19c93285.js | 1.33kb | +0.00b | 7.95s | +2.98s ⚠️ |
| dist/16.51049e2c.js | 1.32kb | +0.00b | 3.17s | +1.68s ⚠️ |
| dist/16.18344f47.js | 1.31kb | +0.00b | 6.97s | +1.62s ⚠️ |
| dist/link.2f4427e6.js | 1.30kb | +0.00b | 7.22s | +1.62s ⚠️ |
| dist/heading6.fdbbdf4d.js | 1.30kb | +0.00b | 7.54s | +1.64s ⚠️ |
| dist/heading3.2113b7a1.js | 1.29kb | +0.00b | 7.54s | +1.64s ⚠️ |
| dist/16.2c4c1846.js | 1.27kb | +0.00b | 6.97s | +1.62s ⚠️ |
| dist/emoji.93ae61d3.js | 1.23kb | +0.00b | 7.10s | +1.63s ⚠️ |
| dist/16.d17ab186.js | 1.23kb | +0.00b | 6.80s | +1.64s ⚠️ |
| dist/16.436f097f.js | 1.21kb | +0.00b | 3.17s | +1.68s ⚠️ |
| dist/16.b4da0acb.js | 1.19kb | +0.00b | 6.80s | +1.64s ⚠️ |
| dist/16.02c68c4f.js | 1.18kb | +0.00b | 6.97s | +1.62s ⚠️ |
| dist/heading5.e195aab1.js | 1.17kb | +0.00b | 7.54s | +1.64s ⚠️ |
| dist/expand.450f3aea.js | 1.16kb | +0.00b | 10.30s | +1.75s ⚠️ |
| dist/media-card-analytics-error-boundary.2323af57.js | 1.15kb | +0.00b | 27.55s | +1.92s ⚠️ |
| dist/16.b46ef690.js | 1.14kb | +0.00b | 3.17s | +1.68s ⚠️ |
| dist/16.4e3b0235.js | 1.14kb | +0.00b | 6.80s | +1.64s ⚠️ |
| dist/16.9ba230ea.js | 1.13kb | +0.00b | 6.61s | +1.65s ⚠️ |
| dist/16.9adb4ec7.js | 1.13kb | +0.00b | 6.80s | +1.64s ⚠️ |
| dist/16.eafe1a11.js | 1.13kb | +0.00b | 7.89s | +2.93s ⚠️ |
| dist/heading2.a2a1be3f.js | 1.11kb | +0.00b | 7.40s | +1.50s ⚠️ |
| dist/16.3920766f.js | 1.10kb | +0.00b | 3.17s | +1.68s ⚠️ |
| dist/component.6600f83e.js | 1.07kb | +0.00b | 5.43s | +801.00ms ⚠️ |
| dist/mention.4d64e059.js | 1.07kb | +0.00b | 7.22s | +1.51s ⚠️ |
| dist/heading4.ec980e93.js | 1.06kb | +0.00b | 7.54s | +1.64s ⚠️ |
| dist/16.afcb09b6.js | 1.05kb | +0.00b | 6.97s | +1.62s ⚠️ |
| dist/layout.52ba15f7.js | 1.05kb | +0.00b | 7.22s | +1.62s ⚠️ |
| dist/divider.16defde6.js | 1.03kb | +0.00b | 7.10s | +1.63s ⚠️ |
| dist/quote.aa25b83c.js | 1.02kb | +0.00b | 7.40s | +1.62s ⚠️ |
| dist/action.9aa72b39.js | 1.00kb | +0.00b | 6.97s | +1.62s ⚠️ |
| dist/decision.20244046.js | 1010.00b | +0.00b | 7.10s | +1.63s ⚠️ |
| dist/panel-warning.97a0e3b3.js | 1008.00b | +0.00b | 7.29s | +1.58s ⚠️ |
| dist/list.c9147a62.js | 978.00b | +0.00b | 7.22s | +1.62s ⚠️ |
| dist/heading1.7d899ba5.js | 976.00b | +0.00b | 7.40s | +1.62s ⚠️ |
| dist/panel-error.8d11e8b7.js | 906.00b | +0.00b | 7.29s | +1.58s ⚠️ |
| dist/panel.a28c56ee.js | 897.00b | +0.00b | 7.29s | +1.52s ⚠️ |
| dist/table.5a4874ee.js | 889.00b | +0.00b | 7.40s | +1.62s ⚠️ |
| dist/panel-success.9b923e58.js | 845.00b | +0.00b | 7.29s | +1.58s ⚠️ |
| dist/panel-note.97a4d541.js | 841.00b | +0.00b | 7.29s | +1.58s ⚠️ |
| dist/media-card-analytics-error-boundary.663369b9.js | 835.00b | +0.00b | 12.16s | +1.80s ⚠️ |
| dist/media-viewer-analytics-error-boundary.0afa7031.js | 755.00b | +0.00b | 14.56s | +1.72s ⚠️ |
| dist/simpleHasher.16c8bf41.js | 528.00b | +0.00b | 12.16s | +1.80s ⚠️ |
| dist/index.html | 119.00b | +0.00b | 7.00s | +964.00ms ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.8016b423.js | 2.72mb | -6.00b 🚀 | 266.00ms | +36.00ms ⚠️ |
| dist/pdfRenderer.15a6a023.js | 1.11mb | +0.00b | 494.00ms | +106.00ms ⚠️ |
| dist/editorView.e06544fb.js | 587.12kb | +0.00b | 570.00ms | +153.00ms ⚠️ |
| dist/popup.6c453e3c.js | 169.96kb | +0.00b | 533.00ms | +129.00ms ⚠️ |
| dist/Toolbar.83ac0b6e.js | 92.34kb | +0.00b | 570.00ms | +153.00ms ⚠️ |
| dist/EmojiPickerComponent.ae601247.js | 75.29kb | +0.00b | 461.00ms | +130.00ms ⚠️ |
| dist/media-viewer.7079364e.js | 72.03kb | +0.00b | 494.00ms | +107.00ms ⚠️ |
| dist/card.645a5c17.js | 55.01kb | +0.00b | 487.00ms | +100.00ms ⚠️ |
| dist/card.0313bac8.js | 52.78kb | +0.00b | 539.00ms | +125.00ms ⚠️ |
| dist/Modal.19ffed32.js | 41.27kb | +0.00b | 285.00ms | +50.00ms ⚠️ |
| dist/component.d7c29980.js | 30.70kb | +0.00b | 266.00ms | +35.00ms ⚠️ |
| dist/component.54080a26.js | 22.50kb | +0.00b | 290.00ms | +50.00ms ⚠️ |
| dist/DatePicker.0df05dac.js | 21.21kb | +0.00b | 461.00ms | +130.00ms ⚠️ |
| dist/EmojiPickerComponent.7dc0ebd0.js | 18.23kb | +0.00b | 461.00ms | +130.00ms ⚠️ |
| dist/dropzone.291ad844.js | 17.34kb | +0.00b | 539.00ms | +129.00ms ⚠️ |
| dist/smartMediaEditor.125a19a7.js | 16.93kb | +0.00b | 570.00ms | +155.00ms ⚠️ |
| dist/js.5733a017.js | 16.40kb | +0.00b | 285.00ms | +50.00ms ⚠️ |
| dist/ui.fee77548.js | 13.97kb | +0.00b | 487.00ms | -191.00ms 🚀 |
| dist/workerHasher.1e706e77.js | 11.75kb | +0.00b | 487.00ms | +123.00ms ⚠️ |
| dist/EmojiTypeAheadComponent.1990d771.js | 8.67kb | +0.00b | 472.00ms | +109.00ms ⚠️ |
| dist/component.77c0f682.js | 6.05kb | +0.00b | 290.00ms | +50.00ms ⚠️ |
| dist/card.383bd9bd.js | 5.61kb | +0.00b | 494.00ms | +107.00ms ⚠️ |
| dist/media-viewer.59c17dad.js | 3.92kb | +0.00b | 494.00ms | +90.00ms ⚠️ |
| dist/EmojiPickerComponent.627aca6b.js | 3.58kb | +0.00b | 472.00ms | +141.00ms ⚠️ |
| dist/index.ac562c80.css | 3.46kb | +0.00b | 630.00ms | +169.00ms ⚠️ |
| dist/png-chunks-extract.a844fdb6.js | 3.41kb | +0.00b | 285.00ms | +49.00ms ⚠️ |
| dist/dropzone.3c67f0a6.js | 3.23kb | +0.00b | 534.00ms | +124.00ms ⚠️ |
| dist/Modal.0304b9ed.js | 3.08kb | +0.00b | 285.00ms | +50.00ms ⚠️ |
| dist/clipboard.da112b6e.js | 2.75kb | +0.00b | 563.00ms | +153.00ms ⚠️ |
| dist/16.290b4a43.js | 2.38kb | +0.00b | 285.00ms | +50.00ms ⚠️ |
| dist/EmojiUploadComponent.d54a1fc6.js | 2.18kb | +0.00b | 472.00ms | +109.00ms ⚠️ |
| dist/card.eea6ad8c.js | 2.03kb | +0.00b | 533.00ms | +129.00ms ⚠️ |
| dist/ResourcedEmojiComponent.abe8596c.js | 2.00kb | +0.00b | 266.00ms | +36.00ms ⚠️ |
| dist/date.231500d2.js | 1.73kb | +0.00b | 349.00ms | +68.00ms ⚠️ |
| dist/16.d16f14ed.js | 1.73kb | +0.00b | 318.00ms | +60.00ms ⚠️ |
| dist/images.80c92f5e.js | 1.67kb | +0.00b | 381.00ms | +100.00ms ⚠️ |
| dist/16.1b6ca79f.js | 1.66kb | +0.00b | 290.00ms | +50.00ms ⚠️ |
| dist/feedback.d5590b34.js | 1.64kb | +0.00b | 459.00ms | +138.00ms ⚠️ |
| dist/browser.75813db0.js | 1.61kb | +0.00b | 563.00ms | +153.00ms ⚠️ |
| dist/workerHasher.6d5b22d7.js | 1.54kb | +0.00b | 472.00ms | +108.00ms ⚠️ |
| dist/workerHasher.fa198233.js | 1.54kb | +0.00b | 539.00ms | +125.00ms ⚠️ |
| dist/list-number.c35f9589.js | 1.46kb | +0.00b | 382.00ms | +85.00ms ⚠️ |
| dist/status.8ab4226b.js | 1.46kb | +0.00b | 429.00ms | +112.00ms ⚠️ |
| dist/16.9ccb5b28.js | 1.38kb | +0.00b | 317.00ms | +73.00ms ⚠️ |
| dist/code.3dd71280.js | 1.38kb | +0.00b | 349.00ms | +83.00ms ⚠️ |
| dist/16.19c93285.js | 1.33kb | +0.00b | 290.00ms | +50.00ms ⚠️ |
| dist/16.c7751e34.js | 1.33kb | +0.00b | 315.00ms | +71.00ms ⚠️ |
| dist/16.51049e2c.js | 1.32kb | +0.00b | 539.00ms | -139.00ms 🚀 |
| dist/16.18344f47.js | 1.31kb | +0.00b | 330.00ms | +64.00ms ⚠️ |
| dist/link.2f4427e6.js | 1.30kb | +0.00b | 381.00ms | +84.00ms ⚠️ |
| dist/heading6.fdbbdf4d.js | 1.30kb | +0.00b | 450.00ms | +129.00ms ⚠️ |
| dist/heading3.2113b7a1.js | 1.29kb | +0.00b | 449.00ms | +129.00ms ⚠️ |
| dist/16.2c4c1846.js | 1.27kb | +0.00b | 330.00ms | +64.00ms ⚠️ |
| dist/emoji.93ae61d3.js | 1.23kb | +0.00b | 350.00ms | +69.00ms ⚠️ |
| dist/16.d17ab186.js | 1.23kb | +0.00b | 318.00ms | +60.00ms ⚠️ |
| dist/16.436f097f.js | 1.21kb | +0.00b | 525.00ms | -153.00ms 🚀 |
| dist/16.b4da0acb.js | 1.19kb | +0.00b | 318.00ms | +60.00ms ⚠️ |
| dist/16.02c68c4f.js | 1.18kb | +0.00b | 330.00ms | +64.00ms ⚠️ |
| dist/heading5.e195aab1.js | 1.17kb | +0.00b | 450.00ms | +129.00ms ⚠️ |
| dist/expand.450f3aea.js | 1.16kb | +0.00b | 459.00ms | +130.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.2323af57.js | 1.15kb | +0.00b | 533.00ms | +129.00ms ⚠️ |
| dist/16.b46ef690.js | 1.14kb | +0.00b | 539.00ms | -139.00ms 🚀 |
| dist/16.4e3b0235.js | 1.14kb | +0.00b | 539.00ms | +282.00ms ⚠️ |
| dist/16.9ba230ea.js | 1.13kb | +0.00b | 314.00ms | +70.00ms ⚠️ |
| dist/16.a37df53b.js | 1.13kb | +0.00b | 315.00ms | +71.00ms ⚠️ |
| dist/16.9adb4ec7.js | 1.13kb | +0.00b | 317.00ms | +73.00ms ⚠️ |
| dist/16.eafe1a11.js | 1.13kb | +0.00b | 290.00ms | +50.00ms ⚠️ |
| dist/heading2.a2a1be3f.js | 1.11kb | +0.00b | 449.00ms | +132.00ms ⚠️ |
| dist/16.3920766f.js | 1.10kb | +0.00b | 525.00ms | -152.00ms 🚀 |
| dist/component.6600f83e.js | 1.07kb | +0.00b | 266.00ms | +36.00ms ⚠️ |
| dist/mention.4d64e059.js | 1.07kb | +0.00b | 410.00ms | +113.00ms ⚠️ |
| dist/heading4.ec980e93.js | 1.06kb | +0.00b | 450.00ms | +130.00ms ⚠️ |
| dist/16.afcb09b6.js | 1.05kb | +0.00b | 330.00ms | +72.00ms ⚠️ |
| dist/layout.52ba15f7.js | 1.05kb | +0.00b | 381.00ms | +84.00ms ⚠️ |
| dist/divider.16defde6.js | 1.03kb | +0.00b | 349.00ms | +68.00ms ⚠️ |
| dist/quote.aa25b83c.js | 1.02kb | +0.00b | 429.00ms | +112.00ms ⚠️ |
| dist/action.9aa72b39.js | 1.00kb | +0.00b | 330.00ms | +64.00ms ⚠️ |
| dist/decision.20244046.js | 1010.00b | +0.00b | 349.00ms | +68.00ms ⚠️ |
| dist/panel-warning.97a0e3b3.js | 1008.00b | +0.00b | 410.00ms | +101.00ms ⚠️ |
| dist/list.c9147a62.js | 978.00b | +0.00b | 382.00ms | +85.00ms ⚠️ |
| dist/heading1.7d899ba5.js | 976.00b | +0.00b | 429.00ms | +112.00ms ⚠️ |
| dist/panel-error.8d11e8b7.js | 906.00b | +0.00b | 410.00ms | +101.00ms ⚠️ |
| dist/panel.a28c56ee.js | 897.00b | +0.00b | 429.00ms | +120.00ms ⚠️ |
| dist/table.5a4874ee.js | 889.00b | +0.00b | 429.00ms | +112.00ms ⚠️ |
| dist/panel-success.9b923e58.js | 845.00b | +0.00b | 410.00ms | +101.00ms ⚠️ |
| dist/panel-note.97a4d541.js | 841.00b | +0.00b | 410.00ms | +101.00ms ⚠️ |
| dist/media-picker-analytics-error-boundary.f8ddfee3.js | 839.00b | +0.00b | 534.00ms | +124.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.663369b9.js | 835.00b | +0.00b | 487.00ms | +100.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.73ba4074.js | 835.00b | +0.00b | 539.00ms | +125.00ms ⚠️ |
| dist/media-viewer-analytics-error-boundary.0afa7031.js | 755.00b | +0.00b | 494.00ms | +90.00ms ⚠️ |
| dist/EmojiPickerComponent.2e54b9c6.js | 535.00b | +0.00b | 472.00ms | +141.00ms ⚠️ |
| dist/simpleHasher.16c8bf41.js | 528.00b | +0.00b | 487.00ms | +123.00ms ⚠️ |
| dist/simpleHasher.a29b2a8e.js | 528.00b | +0.00b | 539.00ms | +124.00ms ⚠️ |
| dist/index.html | 119.00b | +0.00b | 266.00ms | +36.00ms ⚠️ |

</p></details>

<details><summary>Three.js x4 🚨</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | **FAILED** | -0.00ms |
| Cached | **FAILED** | -0.00ms |

#### Cold Bundles

*No bundles found, this is probably a failed build...*
#### Cached Bundles

*No bundles found, this is probably a failed build...*
</p></details>

**[Click here to view a detailed benchmark overview.](https://benchmark.parceljs.org/benchmark/279079992031183373)**

**Comment by devongovett** [comment]
Any idea if this is related to #4498 or #4856?

**Comment by DeMoorJasper** [comment]
@devongovett not sure, would have to try it out

File: packages/core/utils/test/urlJoin.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import urlJoin from '../src/urlJoin';
+
+describe('urlJoin', () => {
+  it('Should join two paths', () => {
+    let joinedUrl = urlJoin('/', './image.jpeg?test=test');
+    assert.equal(joinedUrl, '/image.jpeg?test=test');
+  });
+
+  it('Should join two paths with longer publicUrl', () => {
+    let joinedUrl = urlJoin('/static', './image.jpeg?test=test');
```
Comment (rhendric) [inline packages/core/utils/test/urlJoin.test.js:13]: Think it'd be worth adding another test like this one but for just `'image.jpeg?test=test'`?

Comment (DeMoorJasper) [reply packages/core/utils/test/urlJoin.test.js:13]: sounds like a great idea will add one

**Comment by DeMoorJasper** [comment]
@devongovett just checked it does not, but it does however fix the issue described in #5219

**Comment by mischnic** [comment]
#4856 would be fixed by https://github.com/parcel-bundler/parcel/pull/4842

File: packages/core/core/src/requests/PathRequest.js [hunk 3]
```diff
@@ -151,20 +151,25 @@ export class ResolverRunner {
...
 
+    let queryPart = null;
     if (dependency.isURL) {
       let parsed = URL.parse(filePath);
       if (typeof parsed.pathname !== 'string') {
         throw new Error(`Received URL without a pathname ${filePath}.`);
       }
       filePath = decodeURIComponent(parsed.pathname);
+      if (parsed.query != null) {
+        queryPart = decodeURIComponent(parsed.query);
```
[the remark this answers is not in the document]
Comment (DeMoorJasper) [reply packages/core/core/src/requests/PathRequest.js:162]: Checked the docs and it does exactly that

**Review by devongovett — APPROVED** [review]

**Pull request merged**